program/idl.json

```
@@ -5565,6 +5565,79 @@
           }
         ]
       },
+      {
+        "kind": "instructionNode",
+        "name": "withdrawExcessLamports",
+        "docs": ["Withdraws excess lamports from the account."],
+        "optionalAccountStrategy": "programId",
+        "accounts": [
+          {
+            "kind": "instructionAccountNode",
+            "name": "sourceAccount",
+            "isWritable": true,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["Account holding excess lamports."]
+          },
+          {
+            "kind": "instructionAccountNode",
+            "name": "destinationAccount",
+            "isWritable": true,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["Destination account for withdrawn lamports."]
+          },
+          {
+            "kind": "instructionAccountNode",
+            "name": "authority",
+            "isWritable": false,
+            "isSigner": true,
```

I think this should be the special case `"either"`.

```suggestion
            "isSigner": "either",
```

I know the docs don't make it very clear but if you're passing an array of signers inside `multiSigners`, then the authority is a multisig and should be accepted as "not a signer".